PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,7 +4,6 @@ Make sure these boxes are checked before your pull request is ready to be review
 * [ ] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
 * [ ] pull request is descriptively named with #number reference back to original issue
 * [ ] if possible, multiple commits squashed if they're smaller changes
-* [ ] reviewed/confirmed/tested by another contributor or maintainer
```

Comment:
Github has a good review system now through which we can approve or reject changes. So this is no longer needed now.